src/openforms/authentication/contrib/digid/migrations/0002_auto_20231003_0917.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.20 on 2023-10-02 06:22
+
+from django.db import migrations
+
+from ...utils import update_csp
+
+
+def update_csp(apps, _):
+    """Update csp form-action value if a digid configuration already exists"""
+    DigidConfiguration = apps.get_model("digid_eherkenning", "DigidConfiguration")
```

Comment:
I don't get why this fails in local machine and not in the CI..Here is what I get as an error in my local machine:

`DigidConfiguration = apps.get_model("digid_eherkenning", "DigidConfiguration")
AttributeError: 'DigidConfiguration' object has no attribute 'get_model'`

Reply:
I think because you are importing a function with name `update_csp` and then this function also has name `update_csp`

Reply:
Yes you are right!Thanks, I am pushing an update now.